File: Latihan4/C++/gaji.cpp
#include <iostream>

using namespace std;

int main() {
    string nama, NIK;
    int jk, sp, kendaraan, gajiPokok, uangMakan, Tunjangan, Transport, GajiKotor, Pajak, GajiBersih;

    cout << "Masukan nama : ";
    getline(cin, nama);
    cout << "Masukan NIK : ";
    getline(cin, NIK);
    cout << "Masukan Jenis Kelamin (0=perempuan 1=laki-laki) : ";
    cin >> jk;
    cout << "Masukan Status Pernikahan (0=belum 1=sudah) : ";
    cin >> sp;
    cout << "Masukan Jenis Kendaraan (0=motor 1=mobil) : ";
    cin >> kendaraan;
    cout << "Masukan Gaji Pokok : ";
    cin >> gajiPokok;
    cout << "Masukan Uang Makan : ";
    cin >> uangMakan;

    // menghitung Tunjangan
    if(jk == 1 && sp == 1){
        Tunjangan = 500;
    }else{
        Tunjangan = 0;
    }

    // menghitung transport
    if(kendaraan == 1){
        Transport = 1000;
    }else if(kendaraan == 0){
        Transport = 500;
    }else{
        Transport = 0;
    }

    //gaji
    GajiKotor = gajiPokok + Tunjangan + uangMakan + Transport;
    Pajak = (0.05 * GajiKotor);
    GajiBersih = GajiKotor - Pajak;

    cout << "Tunjangan = " << Tunjangan <<endl;
    cout << "Transportasi = " << Transport <<endl;
    cout << "Gaji Kotor = " << GajiKotor <<endl;
    cout << "Pajak = " << Pajak <<endl;
    cout << "Gaji Bersih = " << GajiBersih <<endl;

    // cout << nama;
    
    return 0;
}

File: Latihan4/Py/gaji.py
# Input Data
NamaKaryawan = input ("Masukkan Nama Karyawan: ")
NIK = input("Masukkan NIK: ")
JenisKelamin = int(input("Masukkan Jenis Kelamin (0=perempuan, 1=laki-laki): "))
StatusPernikahan = int(input("Masukkan Status Pernikahan (0=single, 1=menikah): "))
JenisKendaraan = int(input("Masukkan Jenis Kendaraan (0=motor, 1=mobil): "))
GajiPokok = int(input("Masukkan Gaji Pokok: "))
UangMakan = int(input("Masukkan Uang Makan: "))

# Hitung Tunjangan
if JenisKelamin == 1 and StatusPernikahan == 1:
    Tunjangan = 500
else:
    Tunjangan = 0

# Hitung Transport
if JenisKendaraan == 1:
    Transport = 1000
elif JenisKendaraan == 0:
    Transport = 500
else:
    Transport = 0

# Hitung Gaji Kotor
GajiKotor = GajiPokok + Tunjangan + UangMakan + Transport

# Hitung Pajak
Pajak = 0.05 * GajiKotor

# Hitung Gaji Bersih
GajiBersih = GajiKotor - Pajak

# Output Data
print("Tunjangan:", Tunjangan)
print("Transport:", Transport)
print("Gaji Kotor:", GajiKotor)
print("Pajak:", Pajak)
print("Gaji Bersih:", GajiBersih)
